src/ResourceManager/DataLakeAnalytics/Commands.DataLakeAnalytics/AzureRM.DataLakeAnalytics.psd1

```diff
@@ -0,0 +1,90 @@
+#  
+# Module manifest for module 'Microsoft.Azure.Commands.DataLakeAnalytics'  
```

Comment:
if this is used for help the name should not start with AzureRM. Look at other projects and see how its named